app/src/pages/admin/pluginsPage/pluginsToolbar/actionPanel/uploadCustomBlock/uploadCustomBlock.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { reduxForm } from 'redux-form';
+import classNames from 'classnames/bind';
+import { AttributeListField } from 'components/main/attributeList';
+import { validate } from 'common/utils';
+import { ModalField } from 'components/main/modal';
+import { FieldProvider } from 'components/fields/fieldProvider';
+import { INITIAL_PARAMS_FORM, INITIAL_PARAMS_FIELD_KEY } from '../constants';
+import styles from './uploadCustomBlock.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  initializationParamsLabel: {
+    id: 'UploadCustomBlock.initializationParamsLabel',
+    defaultMessage: 'Initialization parameters',
+  },
+});
+
+@reduxForm({
```

Comment:
I think it's an overkill here. You can pass onChange handler from ImportModal to the custom block and store necessary state there (something like `customState`). Not the best solution in general, but that's all we can do in this case without refactoring.

Reply:
Fixed